swap_meet/vendor.py

```diff
@@ -1,2 +1,105 @@
 class Vendor:
-    pass
+
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory else []
```

Comment:
Good job avoiding the risk of an immutable type in a default parameter!

Reply:
Here's a refresher on why: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/